packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -251,6 +293,74 @@ export class NotifiFrontendClient {
     };
   }
 
+  private async logInWithWeb3(
```

Comment:
I am just wondering the naming role...
It seems like only 'XION` is using this method, but we name it as with `web3` which is kind of general name. Doest it mean we will use it for other upcoming supported chain in the future?